src/server.ts

```diff
@@ -640,11 +640,16 @@ export function createServer(
               }
               ctx.subscriptions[message.id] = operationResult;
 
-              for await (const result of operationResult) {
-                await emit.next(result, execArgs);
+              try {
+                for await (const result of operationResult) {
+                  await emit.next(result, execArgs);
+                }
+                await emit.complete();
+              } catch (err) {
+                await emit.error(err);
```

Comment:
We cannot guarantee here that this error will be an array of `GraphQLError`s. Which the error message requires.
https://github.com/enisdenjo/graphql-ws/blob/1bb9218ad3ee9442442122c1d10910d51951b763/src/message.ts#L54-L58